docs/docs/tour/tour.md

```diff
@@ -0,0 +1,152 @@
+# Tour
+
+Once you've started the DBSP demo, as described in [Get
+Started](../intro), navigate to [localhost:8085](http://localhost:8085)
+in your web browser to view the DBSP user interface.  A status
+dashboard fills most of the home screen, as shown below, with a
+sidebar on the left offering access to tabs for DBSP features:
+
+![DBSP home screen](home.png)
+
+Let's take a quick tour around the options on the sidebar.
+
+## Analytics
+
+![Analytics](analytics.png)
+
+The top group of tabs allows a DBSP user to define the structure of
+data and specify analyses.  Both are these are written as SQL: data
```

Comment:
is "analyses" the best word?

Reply:
> is "analyses" the best word?

What word would you like? This group of tabs in the UI is titled "analytics".

Reply:
Didn't come up with a better one, otherwise I would have posted a suggestion.
"computation" was an alternative, but I am not crazy about it.